### ISSUE TITLE
Feature request: allow to change the User-Agent on the websites

### ISSUE THREAD

**Issue by gorthad1534** [goldendict/goldendict#833]
There is a similar issue #259 that was closed after an option to hide GoldenDict identification had been implemented.
Still, I would really appreciate a fully-featured User-Agent changer.
Some websites look prettier in a mobile version and would fit the GoldenDict interface much better.

**Comment by timofonic** [comment]
I agree this would be interesting. I would also include a random User-Agent feature to avoid banning to GoldenDict.

**Comment by ubless607** [comment]
any update?
